### DISCUSSION TITLE
Preview windows suggestions

### DISCUSSION THREAD

**Discussion by CodeLyokoXtEAM** [Studio-42/elFinder#3313] [Ideas]
In windows 10 Preview window like this and also have button in menu for directly enable it on side (upper left corner in image)
![image](https://user-images.githubusercontent.com/18566890/29883252-5e17422c-8dcd-11e7-8043-37ea41ee1a7d.png)

I think if browser area has enough space, preview window like that.
If not it's better to navbar show it.

Also I found bug that in mobile browser if someone enable preview in navar & hide Navbar,  name columns positions are not correct. Also preview window is hide with it (Not separately showing it)

Edited:-
![screenshot_20170830-220517](https://user-images.githubusercontent.com/18566890/29883908-5dc05ff0-8dcf-11e7-97cb-104efeee6c4c.png)

Also hide navbar & goto icon view then go to list view & show Navbar. It's like this. 

![screenshot_20170830-220805](https://user-images.githubusercontent.com/18566890/29884031-bc953410-8dcf-11e7-8dbb-89ca65b5f9b4.png)

**Comment by CodeLyokoXtEAM** [comment]
Also preview mode volume icons may be have better quality.

**Comment by nao-pon** [comment]
@CodeLyokoXtEAM Problem of table header is a bug. so I'll fix it. The preview area and icons are stacked as future issues.
